src/game.zig

```diff
@@ -911,10 +1024,25 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 		}
 		steppingHeight = @min(steppingHeight, Player.eyePos[2] - Player.eyeBox.min[2]);
 
-		const xMovement = collision.collideOrStep(.client, .x, move[0], Player.super.pos, hitBox, steppingHeight);
+		const slipLimit = 0.00175 * Player.currentFriction;
+		
+		var xMovement = collision.collideOrStep(.client, .x, move[0], Player.super.pos, hitBox, steppingHeight);
 		Player.super.pos += xMovement;
-		const yMovement = collision.collideOrStep(.client, .y, move[1], Player.super.pos, hitBox, steppingHeight);
+		if (KeyBoard.key("crouch").pressed and Player.onGround and @abs(xMovement[0]) < slipLimit) {
```

Comment:
xMovement is the raw movement done in the current frame. Naturally this is fps dependent.
Better compare slipLimit against the velocity to avoid problems.